CHANGELOG/CHANGELOG-0.4.md

```diff
@@ -0,0 +1,7 @@
+## v0.4.0
```

Comment:
v0.3.1

Reply:
I added this to `CHANGELOG-0.4` in the main branch.

https://github.com/kubernetes-sigs/kueue/blob/main/CHANGELOG/CHANGELOG-0.4.md

So fixing this in another PR would be better before we merge this PR into release-0.3.

Reply:
> So fixing this in another PR would be better before we merge this PR into release-0.3.

Ah, this is my misunderstanding.
We need not that.

Reply:
Probably, we can do the same as v0.2.1.

Reply:
I could not modify this branch.
So, I will create another PR.